cli/providers/deploy.go

```diff
@@ -23,11 +26,33 @@ func Deploy(s *service.Service, e *environment.Environment, log *logger.Logger)
 	if err := e.Validate(); err != nil {
 		return err
 	}
-	if err := aws.Deploy(s, e, log); err != nil {
+	switch e.Provider.GetType() {
+	case environment.AWS:
```

Comment:
@rberrelleza I need to refactor this to have a `Provider` interface with common methods between providers of type *aws* or *kubernetes*, but due to lack of time, I will do it in a following PR.